components/camel-as2/camel-as2-api/pom.xml

```diff
@@ -127,7 +127,7 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-javadoc-plugin</artifactId>
                         <configuration>
-                            <additionalparam>-Xdoclint:none</additionalparam>
+                            <doclint>none</doclint>
```

Comment:
Seems to be not related to this PR, please rebuild the whole project

Reply:
Agreed. Thanks. 
I will remove these changes.